Night mode

chobeat opened the pull request.

chobeat committed:
added first css variable

chobeat committed:
added js for night mode

chobeat committed:
added day/night switch

chobeat committed:
extracted some tags

chobeat committed:
added more variables

chobeat committed:
added dark logo

chobeat committed:
made the shrimp green

chobeat committed:
finished tags colors

chobeat committed:
finished colors

chobeat committed:
moved night switch

pizzo00 commented on app/assets/javascripts/night_mode.js:
```diff
@@ -0,0 +1,37 @@
+		}
+
+
+}
```
```
function set_mode(night_mode)
{
    var mode;
    if(night_mode)
    {
        console.log("goodnight");
        mode = 'dark';
    }
    else
    {
        console.log("good morning");
        mode = 'light';
    }
    Cookies.set("night_mode",night_mode)
    document.documentElement.setAttribute('data-theme', mode)
    $("#fs").prop("checked",night_mode)
}
```

pizzo00 replied:
Io lo modificherei così in modo da togliere i due metodi `set_night_mode()` e `set_day_mode()` e ridurre le azioni duplicate.

chobeat replied:
umh, inizialmente quelle funzioni erano diverse tra loro quindi le avevo separate. adesso non ha molto più senso.

chobeat committed:
simplified js

CapacitorSet committed:
Small fixes for deployment

temp

temp

fixed error with RSpec

Fix schema manually (todo: autogenerate so we can be sure it's correct)

chobeat committed:
improved mobile mode

chobeat committed:
styled tags

chobeat committed:
adjusted logo ratio

riggraz committed:
fix typos

riggraz commented:
Ci è sfuggita la pagina /filters

Quanto si è in night mode lo stile lì è una mezza via tra night e light.

chobeat committed:
updated dark logo

riggraz committed:
Set night mode cookie expiry to one year

chobeat committed:
fixed zebra table colors
